build_latest.sh

```diff
@@ -215,16 +215,13 @@ function build_image() {
 	repo=$1; shift;
 	build=$1; shift;
 	btype=$1; shift;
+    local tag=$1; shift;
```

Comment:
whitespace

Reply:
Thanks for pointing it, Will be updating it